### ISSUE TITLE
Boolean columns in original data frame in join

### ISSUE THREAD

**Issue by LudvigOlsen** [biocore-ntnu/pyranges#291]
Very cool package and nice with the givewell mention!

When the original data frame contains a bool column, join fails:

```python
pr.PyRanges(
  pd.DataFrame({
      "Chromosome": "chr1",
      "Start": [0, 10, 20, 30],
      "End": [10, 20, 30, 40]
  })
).join(pr.PyRanges(
  pd.DataFrame({
      "Chromosome": "chr1",
      "Start": [5, 10, 20, 30],
      "End": [20, 20, 30, 40],
      "Site": [True, False, True, True],
  })
), how="right")

```
> Exception: Unknown dtype bool in a column Site

Is there a reason for this or is it a bug?

**Comment by fairliereese** [comment]
Having this issue as well!

**Comment by endrebak** [comment]
I will fix it tomorrow :)

**Comment by endrebak** [comment]
Should be fixed in 0.0.123 which is out now. If something is not as expected, feel free to reopen.

Thanks for reporting.
